database/apikeys.go

```diff
@@ -42,13 +44,32 @@ type (
 	}
 )
 
+// New creates a random new API key.
+func NewAPIKey() APIKey {
+	return APIKey(base32.HexEncoding.WithPadding(base32.NoPadding).EncodeToString(fastrand.Bytes(PubKeySize)))
+}
+
+// Bytes defines the way we decode an API key.
```

Comment:
`nit` I would say it returns the raw byte representation of an APIKey

Reply:
Sounds good. My version sounded awkward to me as well but I couldn't think of better wording on the spot.